# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/embutils/hotfix-clang.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- embutils-0.19/write12.h.vanilla	2023-05-12 20:25:11.468340112 +0200
+++ embutils-0.19/write12.h	2023-05-12 20:25:28.259341325 +0200
@@ -1,6 +1,7 @@
 #ifdef __dietlibc__
 #include <write12.h>
 #else
+#include <unistd.h>
 #include <string.h>
 static inline int __write1(const char* s) {
   write(1,s,strlen(s));
--- embutils-0.19/chown.c.vanilla	2023-05-12 20:24:41.623337957 +0200
+++ embutils-0.19/chown.c	2023-05-12 20:24:54.923338918 +0200
@@ -8,6 +8,7 @@
 #include <grp.h>
 #include <dirent.h>
 #include <sys/stat.h>
+#include <unistd.h>
 #include "write12.h"
 
 #include "error.c"
